src/pcap/cmd/common.go

```diff
@@ -1,125 +0,0 @@
-package cmd
-
-import (
-	"errors"
-	"io"
-	"os"
-	"os/signal"
-	"syscall"
-
-	"github.com/cloudfoundry/pcap-release/src/pcap"
-
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/credentials"
-	"google.golang.org/grpc/credentials/insecure"
-	"google.golang.org/grpc/status"
-)
-
-var zapConfig zap.Config
-
-func init() {
```

Comment:
If you remove the file incl. init-function where the log was configured, you need to configure that again in pcap-agent/main.go and pcap-api/main.go. 

Refer to the commit https://github.com/sap-contributions/pcap-release/commit/dbbdb4107857347a5d699655448cb14e641de28f#diff-6eb5e658f7d5bfc4b5feb25f7207fcac7cd96c459ccf2a6210ca13661481f705R20

Reply:
The logging initialisation is now in `pcap/logger.go`, not removed. It's still loaded automatically for any process that pulls in the `pcap` module.